<ui version="4.0" >
 <class>TimeAveragesForm</class>
 <widget class="QWidget" name="TimeAveragesForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>469</height>
   </rect>
  </property>
  <property name="acceptDrops" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Time averages</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>95</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonAddVar" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/32x32/go-next.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSuppressVar" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/32x32/go-previous.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="3" >
       <widget class="QListView" name="listViewDrop" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QListView" name="listViewDrag" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>95</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="4" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="labelAverage" >
            <property name="text" >
             <string>Label of time average </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lineEditAverage" >
            <property name="toolTip" >
             <string>To give a name for the choosen time average</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelStart" >
            <property name="toolTip" >
             <string>Code_Saturne key word: NTDMOM(IMOM)</string>
            </property>
            <property name="text" >
             <string>Start iteration number
for time average calculation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineEditStart" >
            <property name="toolTip" >
             <string>Code_Saturne key word: NTDMOM(IMOM)</string>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="2" column="0" >
           <widget class="QLabel" name="labelRestart" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IMOOLD(IMOM)</string>
            </property>
            <property name="text" >
             <string>If restart, number in preceding calculation
of the time average to use to initalize
the current selected time average.
</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QComboBox" name="comboBoxIMOOLD" />
          </item>
          <item row="3" column="1" >
           <widget class="QLineEdit" name="lineEditRestart" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IMOOLD(IMOM)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="4" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonAdd" >
            <property name="text" >
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonEdit" >
            <property name="text" >
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete" >
            <property name="text" >
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="4" >
       <widget class="QTreeView" name="treeViewAverage" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="indentation" >
         <number>0</number>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>396</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc" />
 </resources>
 <connections/>
</ui>
